.circleci/config.yml

```diff
@@ -19,53 +23,49 @@ jobs:
           paths:
             - vendor
       - persist_to_workspace:
-          root: /go/src/boscoin.io
+          root: /
           paths:
-            - sebak
+            - go/src/boscoin.io/sebak
+            - home/circleci/.ssh
 
   fmt:
-    docker:
-      - image: circleci/golang:1.10
-    working_directory: /go/src/boscoin.io/sebak
+    <<: *defaults
     steps:
       - attach_workspace:
-          at: /go/src/boscoin.io
+          at: /
       - run:
           name: check formatting
           command: if [[ -n "$(gofmt -l cmd lib)" ]]; then gofmt -l cmd lib; exit 1; fi
 
   test_go1_10:
-    docker:
-      - image: circleci/golang:1.10
-    working_directory: /go/src/boscoin.io/sebak
+    <<: *defaults
     steps:
       - attach_workspace:
-          at: /go/src/boscoin.io
+          at: /
       - run:
           name: run tests
-          command: SEBAK_LOG_HANDLER=null go test -v -timeout 3m ./...
+          command: SEBAK_LOG_HANDLER=null GOCACHE=off go test -v -timeout 3m ./...
 
   test_go1_9:
     docker:
       - image: circleci/golang:1.9
     working_directory: /go/src/boscoin.io/sebak
     steps:
       - attach_workspace:
-          at: /go/src/boscoin.io
+          at: /
       - run:
           name: run tests
-          command: SEBAK_LOG_HANDLER=null go test -v -timeout 3m ./...
+          command: SEBAK_LOG_HANDLER=null GOCACHE=off go test -v -timeout 3m ./...
 
   test_merged:
-    docker:
-      - image: circleci/golang:1.10
-    working_directory: /go/src/boscoin.io/sebak
+    <<: *defaults
     steps:
       - attach_workspace:
-          at: /go/src/boscoin.io
+          at: /
       - run:
           name: Checkout merge commit
           command: |
+            set -ex
```

Comment:
Why is this needed?

Reply:
@outsideris I figure it is useful for debugging :)